gitops-wg/README.MD

```diff
@@ -1,3 +1,89 @@
+
+- Watch or star this repo to see when things change
+- Join the [GitOps Subreddit](https://www.reddit.com/r/GitOps/)
+- Attend a [Working Group or Committee meeting](#meetings)
+- [Open an issue](/../../issues/new) and let us know how you're using GitOps and any important considerations we should include
+- Join `#wg-gitops` on [CNCF Slack](https://slack.cncf.io/)
+- Join the [SIG App Delivery](https://github.com/cncf/sig-app-delivery) mailing list, and watch or participate in topics prefixed with `[gitops-wg]`
+- See [Working Group project boards](https://github.com/orgs/gitops-working-group/projects) for status of work, or for ideas on areas that could use additional participation
+- Volunteer to join [committees](GOVERNANCE.md#committees) and help with projects according to your interest and ability
+
+The Working Group will review all open issues and PRs at our [regular working group meeting](#meetings).
+
+## Meetings
+
+- 🗓 The WG now meets every other Wednesday at [18:00 GMT](https://greenwichmeantime.com/time-gadgets/time-zone-converter/)
+- 👩‍💻 Join using [this Zoom link](https://zoom.us/j/93779536510?pwd=TEFzbGRzREI3MVlkZmluemVkMEhHdz09)
+- ✏️ See [Agenda and Notes](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/) for next and and all previous meetings
+- 💡 Hint: use the [document outline](https://support.google.com/docs/answer/6367684) to help navigate to the right meeting!
+- 📼 Browse the [YouTube playlists](https://www.youtube.com/channel/UCI6iqYuuI4gZuOCZaks5i1g/playlists) for recordings of all previous meetings
+- 📧 Additional committee meetings may be scheduled as needed, and will be announced on the [CNCF App Delivery TAG](https://github.com/cncf/tag-app-delivery) mailing list, prefixed with `[gitops-wg]`
+
+## Timeline
+
+This timeline is a work in progress.
+If you are interested in keeping abreast of any new developments, please see [How to Get Involved](#how-to-get-involved).
+
+| When | What |
+| -- | -- |
+| November 19, 2020 | Amazon, Codefresh, GitHub, Microsoft, and Weaveworks announced the creation of the GitOps Working Group. |
+| Q4 of 2020 | Form the group. Provisional governance bootstrapped. For details, see the initial [GitOps WG Charter](https://docs.google.com/document/d/11EZfvB2FFI837nMmArnyv-wizsIJvc-4_xdgfoUXF4o/view). |
+| Date TBD | Establish a first version of the GitOps Principles |
+| Mid April, 2021 | Clearly established governance. For current status, follow [PR #125](https://github.com/gitops-working-group/gitops-working-group/pull/125). |
+| March 19 - April 16, 2021 | [CFP open](https://docs.google.com/forms/d/e/1FAIpQLSeNahDbiEolx6WZmtxx4L65qmq7pZTX86nQAltq2uC12tCQYg/viewform) for GitOpsCon EU 2021 |
+| May 3, 2021 | GitOpsCon EU 2021. Pre-registration is required. See [Kubecon + CloudNativeCon Co-located events listing](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/program/colocated-events/#gitops-con) for details. |
```

Comment:
These are event specific, I'd get this out of here and put them in EVENTS.md or somewhere better applicable that we refer from here, these events will continually change and we want to limit the need to change the content in this repo I believe

Reply:
Yep agreed, this was from before we had events on https://OpenGitOps.dev

Reply:
Done, and added website tracking issue here: https://github.com/open-gitops/website/issues/37